src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<String> stringArrayList = new ArrayList<>();
+        stringArrayList.add("One");
+        stringArrayList.add("Two");
+        stringArrayList.add("Three");
+
+        stringArrayList.remove(null);

```

Comment:
Calling `remove(null)` on the `ArrayList` might lead to a `NoSuchElementException` if `null` is not present in the list. Ensure that the `ArrayList` implementation can handle this scenario gracefully.